<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="3" skipped="1" tests="4" time="9.309" timestamp="2021-11-22T10:10:10.421247" hostname="Nikolai"><testcase classname="pythontestingframework_execute_tests.tests.test_parameterized" name="test_one_database[test_pre_stage_pool_pass_guest_visits]" file="pythontestingframework_execute_tests\tests\test_parameterized.py" line="41" time="4.166"><failure message="pyodbc.ProgrammingError: ('42000', &quot;[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)&quot;)">values = ['lt_udw_qa_akv', 'lt_udw_qa.pre_stage_pool_pass_GuestVisits.sql', 'lt_udw_qa.stage_pool_pass_GuestVisits.sql', '[Feroz]']

    @pytest.mark.parametrize("values", get_data('test_data_one_db.csv', index=1),
                             ids=get_data('test_data_one_db.csv'))
    def test_one_database(values):
        #   Step 1: Get connection string from command line argument if defined
        # if db_conn == "default db_conn":
        #    db_conn = connections.lt_udw_qa_conn
    
&gt;       db = Database(Database.get_connection(values[0], output_file=False))

pythontestingframework_execute_tests\tests\test_parameterized.py:49: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = &lt;pythontestingframework_execute_tests.db.Database.Database object at 0x0000018EA3ACBFA0&gt;
connection = ('ODBC Driver 17 for SQL Server', 'aeqadb03.database.windows.net', 'lt_udw_qa', 'no', 'InformaticaUser', 'Informatic@1')

    def __init__(self, connection):
    
        if isinstance(connection, str):
            connection = connection.split(',')
        (driver, server, database, trusted_connection, username, password) = connection
        logging.info("Attempting to connect...")
&gt;       self._conn = pyodbc.connect(driver=driver, host=server, database=database,
                                    trusted_connection=trusted_connection, user=username, password=password)
E       pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)")

pythontestingframework_execute_tests\db\Database.py:20: ProgrammingError</failure></testcase><testcase classname="pythontestingframework_execute_tests.tests.test_parameterized" name="test_one_database[test_stage_pool_pass_GuestVisits]" file="pythontestingframework_execute_tests\tests\test_parameterized.py" line="41" time="1.971"><failure message="pyodbc.ProgrammingError: ('42000', &quot;[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)&quot;)">values = ['lt_udw_qa_akv', 'lt_udw_qa.pre_stage_mms_ValAcquisitionType.sql', 'lt_udw_qa.stage_mms_Val_Acquisition_Type.sql', '[Feroz]']

    @pytest.mark.parametrize("values", get_data('test_data_one_db.csv', index=1),
                             ids=get_data('test_data_one_db.csv'))
    def test_one_database(values):
        #   Step 1: Get connection string from command line argument if defined
        # if db_conn == "default db_conn":
        #    db_conn = connections.lt_udw_qa_conn
    
&gt;       db = Database(Database.get_connection(values[0], output_file=False))

pythontestingframework_execute_tests\tests\test_parameterized.py:49: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = &lt;pythontestingframework_execute_tests.db.Database.Database object at 0x0000018EA3C6DEE0&gt;
connection = ('ODBC Driver 17 for SQL Server', 'aeqadb03.database.windows.net', 'lt_udw_qa', 'no', 'InformaticaUser', 'Informatic@1')

    def __init__(self, connection):
    
        if isinstance(connection, str):
            connection = connection.split(',')
        (driver, server, database, trusted_connection, username, password) = connection
        logging.info("Attempting to connect...")
&gt;       self._conn = pyodbc.connect(driver=driver, host=server, database=database,
                                    trusted_connection=trusted_connection, user=username, password=password)
E       pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)")

pythontestingframework_execute_tests\db\Database.py:20: ProgrammingError</failure></testcase><testcase classname="pythontestingframework_execute_tests.tests.test_parameterized" name="test_one_database[test_stage_mms_LTF_Key_Type]" file="pythontestingframework_execute_tests\tests\test_parameterized.py" line="41" time="1.897"><failure message="pyodbc.ProgrammingError: ('42000', &quot;[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)&quot;)">values = ['lt_udw_qa_akv', 'lt_udw_qa.pre_stage_mms_LTF_Key_Type.sql', 'lt_udw_qa.stage_mms_LTF_Key_Type.sql', '[Feroz]']

    @pytest.mark.parametrize("values", get_data('test_data_one_db.csv', index=1),
                             ids=get_data('test_data_one_db.csv'))
    def test_one_database(values):
        #   Step 1: Get connection string from command line argument if defined
        # if db_conn == "default db_conn":
        #    db_conn = connections.lt_udw_qa_conn
    
&gt;       db = Database(Database.get_connection(values[0], output_file=False))

pythontestingframework_execute_tests\tests\test_parameterized.py:49: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = &lt;pythontestingframework_execute_tests.db.Database.Database object at 0x0000018EA3B31E20&gt;
connection = ('ODBC Driver 17 for SQL Server', 'aeqadb03.database.windows.net', 'lt_udw_qa', 'no', 'InformaticaUser', 'Informatic@1')

    def __init__(self, connection):
    
        if isinstance(connection, str):
            connection = connection.split(',')
        (driver, server, database, trusted_connection, username, password) = connection
        logging.info("Attempting to connect...")
&gt;       self._conn = pyodbc.connect(driver=driver, host=server, database=database,
                                    trusted_connection=trusted_connection, user=username, password=password)
E       pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615) (SQLDriverConnect); [42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Cannot open server 'aeqadb03' requested by the login. Client with IP address '217.97.106.107' is not allowed to access the server.  To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.  It may take up to five minutes for this change to take effect. (40615)")

pythontestingframework_execute_tests\db\Database.py:20: ProgrammingError</failure></testcase><testcase classname="pythontestingframework_execute_tests.tests.test_parameterized" name="test_two_databases[values0]" file="pythontestingframework_execute_tests\tests\test_parameterized.py" line="72" time="0.001"><skipped type="pytest.skip" message="got empty parameter set ['values'], function test_two_databases at D:\ISsoft\Life Time\ebi-pythontestingframework-function-qa\pythontestingframework_execute_tests\tests\test_parameterized.py:72">D:\ISsoft\Life Time\ebi-pythontestingframework-function-qa\pythontestingframework_execute_tests\tests\test_parameterized.py:73: got empty parameter set ['values'], function test_two_databases at D:\ISsoft\Life Time\ebi-pythontestingframework-function-qa\pythontestingframework_execute_tests\tests\test_parameterized.py:72</skipped></testcase></testsuite></testsuites>